vendor/bitbucket.org/ww/goautoneg/.hg_archival.txt

```diff
@@ -1,6 +0,0 @@
-repo: 848b351341922ce39becda978778724d5b58dbca
-node: 75cd24fc2f2c2a2088577d12123ddee5f54e0675
-branch: default
-latesttag: null
-latesttagdistance: 5
-changessincelatesttag: 5
```

Comment:
what's going on here?  do you have an old dep? or did the person who created this have an old dep?

Reply:
> what's going on here? do you have an old dep? or did the person who created this have an old dep?

no idea.

Reply:
glide does bad stuff - it doesn't store `.git` but it does store the internal metadata for mercurial. given different mercurial store different metadata version, diffing gets impossible - we are fixing this post `glide update` in 
https://github.com/openshift/library-go/blob/b25329afa1ae9d7dfcd7c3d8d19940d7bb87afdd/alpha-build-machinery/make/targets/openshift/deps.mk#L16